### DISCUSSION TITLE
No ILS Signal with the A320 at Addon Airports

### DISCUSSION THREAD

**Discussion by Emerica1208** [flybywiresim/aircraft#2401] [Help]
Hello people,I recently noticed that I can't receive ILS signal at various addon airports with the A320Neo. If I fly to airports for which I do not have an addon, the ILS works without problems.
Is this a problem of the FlyByWire Airbus or is it a general problem and a bug in my MSFS?

Thank you and best regards

**Comment by MapleNAV-TCASFAULT** [comment]
Did you check your RAD NAV page of the MCDU, if the ILS freq and CRS is missing, you must fill it out!

**Comment by Emerica1208** [comment]
I know that this is a workaround for the problem, but isn't it normal, that the A320 will catch the ILS by it self?

**Reply by MapleNAV-TCASFAULT** [reply]
It is supposed to, but due to some changes with the ILS to match the airport scenery it may cause the MCDU to not retrieve the ILS info.

**Comment by mga010** [comment]
I had the same issue yesterday in LSGG.  I could enter the frequency, but not the course. And the ILS was not received. I had the latest development version.
